### PR TITLE
Azure: Add goimports and golint to the base test image

### PR DESCRIPTION
**Pull request by asalkeld**

### CODE REVIEW

**Comment by mjudeikis** [comment]
/lgtm 
follow-up with low priority. I dont like those go get... 
https://github.com/openshift/openshift-azure/issues/516

File: projects/azure/base-images/test-base.yaml [hunk 1]
```diff
@@ -48,6 +48,8 @@ items:
         # TODO: I should not be here. Remove me.
         RUN go get github.com/go-bindata/go-bindata && \
             go get golang.org/x/tools/cmd/goyacc && \
+            go get golang.org/x/tools/cmd/goimports && \
+            go get golang.org/x/lint/golint && \
```
Comment (kargakis) [inline projects/azure/base-images/test-base.yaml]: We don't need golint in here; if you want to lint PRs you can always comment `/lint` on PRs

**Comment by kargakis** [comment]
What's the goal here? Are we going to use goimports in CI? golint is not needed.

**Comment by asalkeld** [comment]
@kargakis https://github.com/openshift/openshift-azure/pull/514

**Comment by asalkeld** [comment]
since hardly anyone does "/lint" I think it's worthwhile adding on every make verify

**Comment by kargakis** [comment]
> since hardly anyone does "/lint" I think it's worthwhile adding on every make verify

Historically lint has caused issues in our queue because it can get very opinionated that's why we have avoided blocking on it.

**Comment by asalkeld** [comment]
golint removed

**Pull request merged**